src/commands/version/index.ts

```diff
@@ -69,10 +69,9 @@ export default class VersionCmd extends Command {
 		const { flags: options } = await this.parse(VersionCmd);
 		const versions: JsonVersions = {
 			'balena-cli': (await import('../../../package.json')).version,
-			'Node.js':
-				process.version && process.version.startsWith('v')
-					? process.version.slice(1)
-					: process.version,
+			'Node.js': process.version.startsWith('v')
```

Comment:
not `process.version?.startsWith('v')`

Reply:
Nope. process.version is a mandatory property from nodejs runtime.